src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx

```diff
@@ -3105,6 +3105,16 @@ For more details, see our documentation about [APM logs in context](/docs/apm/ne
 </applicationLogging>
 ```
 
+These features can also be configured via environment variables:
+
+```
+NEW_RELIC_APPLICATION_LOGGING_ENABLED="true"
+NEW_RELIC_APPLICATION_LOGGING_METRICS_ENABLED="true"
+NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED="false"
+NEW_RELIC_APPLICATION_LOGGING_FORWARDING_MAX_SAMPLES_STORED="10000"
+NEW_RELIC_APPLICATION_LOGGING_LOCAL_DECORATING_ENABLED="false"
+```
+
 <Callout variant="important">
```

Comment:
you placed your addition in between the thing above and this callout so I'm curious if you have an opinion about where that callout should live.

Reply:
All of the config options under `<applicationLogging>` are only available in 9.7.0+, so at the very least we should change the wording to be "These configuration options are only available in the .NET agent v9.7.0 or higher."

I could see it going before the XML config examples.  If you agree I can make the changes myself.

Reply:
I'll get it. Thanks!